src/lib/network/network_autoconfiguration.rb

```diff
@@ -90,15 +90,7 @@ def configure_virtuals
     # Propose DNS and Hostname setup
     def configure_dns
```

Comment:
it seems to be completely useless now ;-) However I'm not sure if removing ```ProposeHostname``` is correct

Reply:
Why?

Reply:
I've restored the method (now it is called #propose_hostname) so we are hopefully on the safe side :)